.github/workflows/create-release-pr.yaml

```
@@ -11,10 +11,35 @@ jobs:
     name: Create Release PR
     runs-on: ubuntu-latest
     steps:
+      - name: Check if a branch is whitelisted and maintained
+        id: branch-check
+        run: |
+          BRANCH_NAME="${GITHUB_REF#refs/heads/}"
+          WHITELISTED_BRANCHES=("main" "release/*")
```

```suggestion
          WHITELISTED_BRANCHES=("main" "releases/*")
```
we use `releases/*` as the convention in existing repos (and our rulesets)

---

Address this?